### DISCUSSION TITLE
建议添加“复制当前视频时间地址”的功能

### DISCUSSION THREAD

**Discussion by G4Y8u9** [the1812/Bilibili-Evolved#1465] [功能建议 / Ideas]
和朋友分享视频的时候很好用。。每次自己都是手动算好时间然后手打“?p=多少&t=多少”

非常感谢！

**Comment by the1812** [comment]
可以直接在顶栏历史面板里复制链接

**Reply by Geeyun-JY3** [reply]
其实有考虑过把这个功能做进脚本吗（好像有不少有这功能的脚本，复制的方法也千奇百怪）？
（还有就是这个 discussion 居然是第一个提的？）

**Reply by the1812** [reply]
或许可以塞到附加功能里, 复制 av 号的地方

**Reply by Geeyun-JY3** [reply]
会不会太深了，YouTube、优酷、腾讯都是塞播放器右键菜单的，塞附加功能可能还得暂停

**Reply by the1812** [reply]
右键菜单啊, 那可以等我写出右键菜单扩展的API, 我顺便把之前希望恢复右键倍速/循环之类的也做了

**Reply by Geeyun-JY3** [reply]
[哔哩哔哩播放器增强](https://github.com/fython/userscript-enhance-bilibili-player) 就是这么做的，还能录视频🌚

**Reply by Geeyun-JY3** [reply]
要做的话我这还能补充些信息：B 站还有个 `?start_progress=[微秒数]` 的跳转参数，比 `t=` 的更精确。
之前在这个 [活动页](https://www.bilibili.com/blackboard/topic/activity-NZd9XjWOd.html) 发现的。当时测试只能在番剧页生效，现在绝大部分视频播放页面下都可用了。
https://github.com/fython/userscript-enhance-bilibili-player/issues/26

**Reply by Geeyun-JY3** [reply]
今天确认，`?start_progress=[毫秒数]` 这个跳转参数已经失效（应该有好一段时间了），令人感叹。
（才发现之前写错了，应该是毫秒）

**Comment by Geeyun-JY3** [comment]
这个功能的脚本挺多的，着急可以先用这个：https://github.com/fython/userscript-enhance-bilibili-player

联系：https://github.com/the1812/Bilibili-Evolved/issues/546#issuecomment-632648624

**Reply by G4Y8u9** [reply]
哇 非常感谢！

**Comment by Geeyun-JY3** [comment]
破站终于把这功能做出来了，落后全网不知多少年：`复制视频地址（精准空降）`
![复制视频地址（精准空降）](https://user-images.githubusercontent.com/49396421/138223872-8cf81568-79cf-4663-a363-f3679feb6492.jpg)

**Reply by Geeyun-JY3** [reply]
服了，这功能做出来一年多，复制出来的链接居然还是没有分 P 参数😅

**Reply by Geeyun-JY3** [reply]
再一年后回头看，复制出来的链接已经有分 P 参数了。
